src/LuaPrinter.ts

```diff
@@ -582,13 +588,20 @@ export class LuaPrinter {
             } else {
                 chunks.push("\n");
                 this.pushIndent();
-                expression.fields.forEach(f => chunks.push(this.indent(), this.printTableFieldExpression(f), ",\n"));
+                expression.fields.forEach(
+                    f => {
+                        const fieldExpression = this.printTableFieldExpression(f);
+                        fieldExpression.prepend(this.indent()); // make indent part of the field expression
+                        chunks.push(fieldExpression);
```

Comment:
The indent is still a separate chunk here, but it's being put on the field expression SourceNode, instead of being part of the table expression SourceNode. This way the indent doesn't get mapped back to the line that starts the table expression.